src/main/java/uk/gov/justice/hmpps/probationteams/config/ClientTrackingTelemetryModule.kt

```diff
@@ -1,33 +1,24 @@
-package uk.gov.justice.hmpps.probationteams.config
+package uk.gov.justice.digital.hmpps.probationteams.config
```

Comment:
package change to `digital` - rest don't seem to be under digital

Reply:
moved this